ast/parser_test.go

```diff
@@ -1542,7 +1542,7 @@ func TestRule(t *testing.T) {
 				Value:     IntNumberTerm(2),
 				Assign:    true,
 			},
-			Body: NewBody(NewExpr(BooleanTerm(true))),
+			//	Body: NewBody(NewExpr(BooleanTerm(true))),
```

Comment:
Why is this commented out? Same comment below as well.

Reply:
Thanks for the comment.

This is the policy `x := 1 { false } else := 2`. 
The previous version of parser adds a body ` { true } ` after the `else := 2` statement, resulting in the expected result: `x := 1 { false } else := 2 { true }

The new version of parser does not add anything, so the new expected result is `x := 1 { false } else := 2`.

The test case is updated accordingly, 'Body: NewBody(NewExpr(BooleanTerm(true))),' is removed.

Reply:
Similarly in test case `assertParseRule(t, "wildcard in else args",` ```f(_) { true } else := false```

The previous expected result is f(_) = true { true } else = false { true }.
The new expected result is f(_) = true { true } else = false.

So 'Body: NewBody(NewExpr(BooleanTerm(true))),' is removed accordingly.